regolith/helpers/l_contactshelper.py

```diff
@@ -22,6 +22,11 @@
 
 
 def subparser(subpi):
+    subpi.add_argument(
```

Comment:
I think it would be better if this is like  cmd not a flag... so one would type:
```
regolith helper l_contacts run
```

Reply:
Done